### ISSUE TITLE
Defined Measures - unable to list $0.00

### ISSUE THREAD

**Issue by BrianaParrBaker** [fishbaugher/WAweb-Issues#44]
Unable to list $0.00 in a detail. It says they are incomplete.
![Screenshot 2024-03-08 100505](https://github.com/fishbaugher/WAweb-Issues/assets/162485966/24a0c9b8-1d39-45ff-b512-a69ae338e58d)

**Comment by fishbaugher** [comment]
Confirmed.  In v8, place holder cost detail records with zero cost can be added to Defined Measures which brings them forward (with zero cost) through the audit (by reference) then into the work order (by copy)

Missing from v10

**Comment by fishbaugher** [comment]
Related to #59.  Same basic problem with Defined Measures not carrying detail through audit to the work order measure detail level via the Itemized Cost in the audit.

**Comment by eckmanwe** [comment]
We disagree that this is related to #59 
Likewise, there is a simple workaround: use $0.01 or if a goal is to trigger detail entry by the user, then consider $9999.

For additional consideration by the WA Advisory Group - Please provide viable use cases where $0 cost for a Defined Measure material/labor cost line item makes sense.

**Comment by BrianaParrBaker** [comment]
One example is for a furnace or water heater replacement that was awarded at a set, procurement price (as opposed to a bid amount). We include line items for mechanical permits, electrical subcontracting, and electrical permits, but put the total procurement price in only the material and labor line items. We have found that putting estimated values in the permit and subcontractor lines is confusing for contractors, so it is better to put $0 in those lines and allow the contractor to put the actual amounts into the actual column. 
Here is an example in old WA:
![procurement](https://github.com/fishbaugher/WAweb-Issues/assets/162485966/5bd304d7-1270-467f-af93-1776535ac6ce)

**Comment by fishbaugher** [comment]
It would seem that the quantity = 1 and estimate unit cost = $0.01 should be a decent work around.

**Comment by BrianaParrBaker** [comment]
I disagree that an estimated unit cost of $0.01 is a decent workaround. It does not avoid the issue of clear material and labor breakdowns of procurement prices. I can make it work if necessary, but "Workaround is Poor" is more accurate for our agency than "Workaround is Good".

**Comment by fishbaugher** [comment]
Perhaps I don't understand.  If qty 1 and price 0.01 is used in lieu of 0 and 0, you still have the same set of detail records that should carry forward to the work order.  The contractor would delete the detail records that don't apply, and change the qty 1 and price 0.01 for those details that do apply.  Yes?

**Comment by BrianaParrBaker** [comment]
There is still the same problem of clear material and labor breakdowns of procurement prices. Either the measure subtotal needs to be off, or the estimated material or labor amount needs to be adjusted to account for the additional $.01 (in the example below, $.03). As I mentioned, we have found that it is confusing for contractors if we unnecessary adjust the estimated material and labor breakdowns from the set procurement amounts. 

Measure subtotal off:
![03](https://github.com/fishbaugher/WAweb-Issues/assets/162485966/41354a1c-dc53-4fe3-a72d-fb3303773776)

Adjusted material and labor amounts so they no longer match set procurement amounts: 
![98](https://github.com/fishbaugher/WAweb-Issues/assets/162485966/d6e5370f-8930-4528-982e-68cba20f7c13)

**Comment by fishbaugher** [comment]
Thank you Briana, that makes the need for zeros clear.

**Comment by fishbaugher** [comment]
Another point.. the software _does_ allow zeros for Measure Cost details:

<img width="754" alt="Untitled" src="https://github.com/fishbaugher/WAweb-Issues/assets/479175/712a8a9c-4e2d-4344-8946-de91589eb091">

So, for consistency, it seems to make sense to allow it for Defined Measures as well.
